Community-Committee-Charter.md

```diff
@@ -4,65 +4,56 @@
 
 Changes to CommComm membership should be posted in the agenda, and may be suggested as any other agenda item.
 
 No more than one-fourth of the CommComm members may be affiliated with the same employer or leadership of a community/ecosystem organization. If removal or resignation of a CommComm member, or a change of employment by a CommComm member, creates a situation where more than one-fourth of the CommComm membership shares an employer, then the situation must be immediately remedied by the resignation or removal of one or more CommComm members affiliated with the over-represented employer(s).
 
-The CommComm members shall consist of active members of Community Projects and the two Individual Membership Directors.
+The CommComm members shall consist of active members of Community Projects.
 
 The CommComm may, at its discretion, invite any number of non-voting Guests to participate in the public portion of CommComm discussions and meetings.
 
-The CommComm shall meet regularly using tools that enable participation by the community (e.g. weekly on a Google Hangout On Air, or through any other appropriate means selected by the CommComm). The meeting shall be directed by the Individual Membership Directors. Minutes or an appropriate recording shall be taken and made available to the community through accessible public postings.
+The CommComm shall meet regularly using tools that enable participation by the community (e.g. weekly on a Google Hangout On Air, or through any other appropriate means selected by the CommComm). Minutes or an appropriate recording shall be taken and made available to the community through accessible public postings.
 
 CommComm members are expected to regularly participate in CommComm activities.
 
-In the case where an individual CommComm member -- within any three month period -- attends
-fewer than 25% of the regularly scheduled meetings, does not participate in CommComm
-discussions, and does not participate in CommComm votes, the member shall be automatically
-removed from the CommComm. The member may be invited to continue attending CommComm meetings
-as a Guest.
+In the case where an individual CommComm member -- within any three month period -- attends fewer than 25% of the regularly scheduled meetings, does not participate in CommComm discussions, and does not participate in CommComm votes, the member shall be automatically removed from the CommComm. The member may be invited to continue attending CommComm meetings as a Guest.
 
-## Section 5. Responsibilities of the CommComm.
+## Section 4. Responsibilities of the CommComm.
 
 Subject to such policies as may be set by the Board, the CommComm is responsible for all cultural development and outreach within the Node.js Foundation, including:
 
 - Outreach to community organizations
 - Documentation of community organizations
-- Cultural direction of Node.js Foundation
-- Overseeing the Inclusivity WG
```

Comment:
Why we removed "Inclusivity WG"? Because this WG isn't active anymore?

Reply:
At the time of the inception of the CommComm, the Inclusivity WG existed and the CommComm's structure was largely built out by folks from that WG. At the time of writing this document, I can only assume it was future-looking based off the current state of things.

There's a few elements of this that don't match now:
- There is no concept of a "WG" in the CommComm
- The Inclusivity WG does not exist in the same state, and I am relatively certain that there will likely not be another WG that brandishes the exact same name (being explicit: this isn't to say there won't be a WG or initiative that aims to address the same problem space) 
- There is no precedent for including specific WGs or Initiatives in a charter, and it actually _limits_ the scope of what we can do rather than expanding that scope

Reply:
I agree with the last point you made there, mentioning specifics in a charter limits the scope of work for CommComm and calls for unnecessary changes.

Reply:
Makes sense 👍